docs/content/en/docs/operator-manual/piped/configuration-reference.md

```diff
@@ -36,8 +36,8 @@ spec:
 
 | Field | Type | Description | Required |
 |-|-|-|-|
-| username | string | The username that will be configured for `git` user. | No |
-| email | string | The email that will be configured for `git` user. | No |
+| username | string | The username that will be configured for `git` user. Default is `piped`. | No |
+| email | string | The email that will be configured for `git` user. Default is `pipecd.dev@gmail.com`. | No |
```

Comment:
I think we should make these default values to "" ( empty string ), users should be aware of which GitHub account should be set before using this feature ( without GitHub account be set, error no account set should be raised instead of not existed account or something else ) 🤔

Reply:
To my understanding, a git user and a GitHub user are totally unrelated. I'd say it just uniquely identifies a user in a commit within a single project.

Reply:
my bad, I mean git user 🙏

Reply:
By showing the default user in the docs, I think users can be aware of which git user will be used, but what do you think?

Reply:
I got what you wanted to say. Users should know what https://github.com/pipecd-bot will be used, right?

Reply:
yeh, sorry if I guess wrong. I thought `pipecd-bot` is an internal use (our team's bot), and the normal users should/must aware of which bot would be used in their environment.

Reply:
what I thought was the behavior of piped in case of unset git account. Currently, looks like we will use the default git account and try to commit first, I think the error `unset git account` is better suit for this situation.

Reply:
Thanks for giving good points of view. I think it's not a problem that pipecd-bot will be used by end-users. So let me add a description of it in another PR. I'd say it's enough but how about you?

Reply:
My browser couldn't be refreshed.

>what I thought was the behavior of piped in case of unset git account. Currently, looks like we will use the default git account and try to commit first, I think the error unset git account is better suit for this state.

Again, I think it's not a problem that pipecd-bot will be used by end-users. So it's enough to provide pipecd-bot as a default account to reduce the need to set username and email.

Reply:
> I think it's not a problem that pipecd-bot will be used by end-users

If the pipecd-bot could be shared externally, then it's okay to me 👍 Thanks for correcting my viewpoint 🙏

Reply:
> My browser couldn't be refreshed.

me too 🤣

Reply:
Thank you for giving the opportunity to re-think about its lisk!

Reply:
haha, tough to do a conversation on the Github 😭